2주차/11047.py

```diff
@@ -0,0 +1,14 @@
+N,K = map(int,input().split())
+money=[]
+for _ in range(N):
+    money.append(int(input()))
```

Comment:
```suggestion
money=[int(input()) for _ in range(N)]
```
이런 입력 받을때는 리스트 컴프리헨션 사용해보세요